index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
Do we know what units the Android API uses? Whatever we choose we should be specific about it, and not leave it up to devices. This PR says milliliters which sounds fine to me, I'm just not a domain expert.